### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by bhaskarbabu520** [mRemoteNG/mRemoteNG#2590] [General]
Hi,
When can we expect a new release?
Thanks

**Comment by nelgin** [comment]
Never. This appears to be a dead project.

**Comment by CreoleBrown** [comment]
Its definitely dead, there are many other projects out there better, this one is dead and buried

**Reply by real-napster** [reply]
Can you name one other?

**Reply by QuantMad** [reply]
Same question

**Reply by bhaskarbabu520** [reply]
This is a great tool, but not approved to use by my company

**Reply by nelgin** [reply]
@rmack00 and how much are you paying for a license?

**Reply by Kvarkas** [reply]
I dont mind you could use any software you like, but unfortunately they dont pay us for advertise so any mention of other products will be deleted

**Comment by bhaskarbabu520** [comment]
Could you please let me know the free alternative to mremoteng which offers same or better features?

**Reply by Kvarkas** [reply]
where is none :)

**Reply by nelgin** [reply]
Unfortunately, this is as good as it gets if you don't want to dip your hand into your pocket.

**Comment by Kvarkas** [comment]
same people keep crying without committing any line of code! so funny :)

**Reply by nelgin** [reply]
Boohoohoo. Not my project.  I don't do windows coding.  I assist in projects thank you very much.

**Reply by Kvarkas** [reply]
Anyway you are welcome to comment but don't expect from one maintainer 8h work on this project everyday, however belter later than never ;)

**Comment by Kvarkas** [comment]
> Hi, When can we expect a new release? Thanks

Soon, at this moment a huge refactoring is happening related to security regulations.

**Reply by bhaskarbabu520** [reply]
Thank you, waiting for it, This is a great tool make life much easier.

**Reply by bhaskarbabu520** [reply]
Hi,
Can you please let me know if you have any update. 
Thank you!

**Reply by bhaskarbabu520** [reply]
Can you please update on the tentative release date?

**Comment by mcdonamw** [comment]
Understanding the refactor is going to take some time.  Any chance this is going to go 64bit?  I'm constantly running into virtual memory errors with this 32bit version hitting 2gb memory after just connecting to 5-6 RDP sessions.

**Reply by Kvarkas** [reply]
will try to switch to full 64bits once we migrate to .net 8

**Comment by krzys128** [comment]
I wonder about some Christmas gift like  ... a new release ?   Any chance ?  For me almost everything works fine . Maybe would be nice to save options.  If not this year it would be good to write a note on main page of project that You @Kvarkas are alive, you working hard and expect to share your work with us in ..... weeks/months.   Or something similar. 

I am not complaining, not hurring you up  -  just ask :)  

loyal user of nRemoteNG 
K.

**Comment by Kvarkas** [comment]
Yes working in a rush! this time of the year all want to do as much as posible - trying to be helpfull :) at this moment still working on improvements for a security issue in regards passwords in memory, as solution decide to modify a logic a bit, so instead of load all into memory, will store such in local connection db (that will be encrypted per machine and could be password protected for etra paranoid users :) at same time each connection will got its own ID and on connection request from a click - data and credentials will be fetch just to open connection! So time of credentials will be disposed in memory (even via secure string!) will be super small! So far some more testing should happening to be shure its wokring properly! Stay tuned! :)

**Reply by KarunSaikia** [reply]
I'm following up to inquire if a new version will be available soon. Thank you

**Reply by mdell-seradex** [reply]
I hope this is progressing well.
Any thoughts as to when you **hope** to release the new version.  I will not hold you to that date, or even near it, and no one else should either.  Just was hoping for some indication on this.
I would like to offer help, but I spend too much time programming for my job. 🙂
I have contributed little bits to others over time, but I typically don't have much time to spare.

**Comment by simonaubertbd** [comment]
@Kvarkas Hello. I noticed you committed a few things these last days. Hope that's going well and thank you for your efforts.

**Comment by Kvarkas** [comment]
not like full NB release (with install) but good one on new rails :) still 2 main things to implement before - storage and maintaining configs and on same basis credentials secure handlening > https://github.com/orgs/mRemoteNG/discussions/2680 <
